src/Features/Core/Portable/ConvertToInterpolatedString/AbstractConvertPlaceholderToInterpolatedStringRefactoringProvider.cs

```diff
@@ -35,18 +35,22 @@ public override async Task ComputeRefactoringsAsync(CodeRefactoringContext conte
             var semanticModel = await document.GetRequiredSemanticModelAsync(cancellationToken).ConfigureAwait(false);
 
             var stringType = semanticModel.Compilation.GetSpecialType(SpecialType.System_String);
-            if (stringType == null)
+
+            if (stringType is null)
             {
                 return;
             }
 
-            var formatMethods = stringType
-                .GetMembers(nameof(string.Format))
-                .OfType<IMethodSymbol>()
-                .Where(ShouldIncludeFormatMethod)
-                .ToImmutableArray();
+            var consoleType = semanticModel.Compilation.GetTypeByMetadataName(typeof(Console).FullName);
```

Comment:
Should we eventually support all methods mentioned in [composite-formatting](https://docs.microsoft.com/dotnet/standard/base-types/formatting-types#composite-formatting)?

Reply:
maybe. The intention of this PR was to open that possibility up easily without adding them, since the original issue and design only mentioned console.

Reply:
yea, I'm really thinking there should be some sort of abstraction here for all of that...

Reply:
Updated included ones in this PR